File: 1-Iniciante/ex1071.py
#entrada
i = int(input())
f = int(input())
s = 0

#processamento
if i > f:
    aux = i
    i = f
    f = aux

for n in range(i + 1, f, 1):
    if n % 2 == 1:
        s += n

#saida
print(s)

File: 1-Iniciante/ex1071.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8');
let [x, y] = input.split('\n').map(v => parseInt(v));

// processamento
if(x > y){
    let temp = x;
    x = y;
    y = temp;
}

if(x % 2 === 0) x++;
else x += 2;

let soma = 0;
for(let i = x; i < y; i += 2) soma += i;

// saida
console.log(soma);